### ISSUE TITLE
Could not install GPG key error

### ISSUE THREAD

**Issue by mundodisco8** [headmelted/codebuilds#71]
Hi,

I'm trying to install VS Studio on a clean Raspbian install on a Raspberry Pi 3, and I'm getting a problem with the GPG Key.

``` bash
root@raspberrypi:/home/pi# . <( wget -O - https://code.headmelted.com/installers/apt.sh )
--2019-03-20 09:42:47--  https://code.headmelted.com/installers/apt.sh
Resolving code.headmelted.com (code.headmelted.com)... 104.27.187.80, 104.27.186.80, 2606:4700:30::681b:bb50, ...
Connecting to code.headmelted.com (code.headmelted.com)|104.27.187.80|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-sh]
Saving to: ‘STDOUT’

-                                         [ <=>                                                                    ]   2.29K  --.-KB/s    in 0s      

2019-03-20 09:42:47 (6.47 MB/s) - written to stdout [2349]

Detecting architecture...
Ensuring curl is installed
Reading package lists... Done
Building dependency tree       
Reading state information... Done
curl is already the newest version (7.52.1-5+deb9u9).
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
Architecture detected as armv7l...
Retrieving GPG key [headmelted] (https://packagecloud.io/headmelted/codebuilds/gpgkey)...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
gpg: no valid OpenPGP data found.
```

I managed to work around it by downloading the script, and uncommenting the line with the `--allow-unauthenticated` bit.

``` bash
apt-get install -t ${repo_name} -y ${code_executable_name};
#apt-get install -t ${repo_name} -y --allow-unauthenticated ${code_executable_name};
````

The funny thing is that I swear this worked yesterday, with an old Raspbian install. I've tried #63 but that doesn't help. Any clues?

**Comment by magic-k** [comment]
I use 
```
wget https://packagecloud.io/headmelted/codebuilds/gpgkey
sudo apt-key add gpgkey
```

**Comment by kobzar** [comment]
You can use this for install it manual in chroot
https://packagecloud.io/headmelted/codebuilds/install

**Comment by ZimbiX** [comment]
This issue can now be closed as it's been resolved in 38e4efe.
